src/EA.Weee.Web/Areas/AatfReturn/Controllers/ObligatedValuesCopyPasteController.cs

```diff
@@ -64,6 +69,11 @@ public virtual async Task<ActionResult> Index(ObligatedValuesCopyPasteViewModel
                 }
             }
 
+            if (viewModel.Type == ObligatedType.Reused)
+            {
+                return await Task.Run<ActionResult>(() => AatfRedirect.ObligatedReused(viewModel.ReturnId, viewModel.AatfId));
```

Comment:
Add unit tests to test for this redirect when reused

Reply:
Added tests for both.